cmd/node/operationModes.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import "fmt"
+
+const (
+	operationModeImportDb           = "import-db"
+	operationModeFullArchive        = "full-archive"
+	operationModeDbLookupExtension  = "db-lookup-extension"
+	operationModeHistoricalBalances = "historical-balances"
+)
+
+func checkOperationModes(modes []string) error {
+	// db lookup extension and historical balances
```

Comment:
For reviewers: think about other combinations of modes that can lead to unexpected configuration. For now, I've checked modes that altered the same fields

Reply:
added lite observer mode. it won't be compatible with historical balances and full archive, since the old data is removed